File: Python/baekjoon/단계별로 풀어보기/문자열/10809_알파벳 찾기.py
# https://www.acmicpc.net/problem/10809

li = [-1] * 26
for i, s in enumerate(input()):
    if li[ord(s) - 97] == -1:
        li[ord(s) - 97] = i
print(' '.join(map(str, li)))

File: C#/Baekjoon/문자열/10809_알파벳 찾기.cs
//https://www.acmicpc.net/problem/10809

int[] a = new int[26];
for (int i = 0; i < 26; i++) a[i] = -1;
string s = Console.ReadLine();
int cnt = 0;
foreach (char c in s)
{
    if (a[c - 97] == -1)
    {
        a[c - 97] = cnt;
    }
    cnt++;
}
Console.Write(string.Join(" ", a));

//string s = Console.ReadLine();
//for (char c = 'a'; c < '{'; c++)
//{
//    Console.Write(s.IndexOf(c) + " ");  // Indexof 없으면 -1 반환
//}